### PR TITLE
Create new food category for ASRS.

### PR DESCRIPTION
**Pull request by Huffie56**
## About The Pull Request

1-creating a new category in _supply_packs.dm
2-creating a new file named food.dm
3-add new crates with (potato wheat tomato)*20 meat and fish  *5 and a crate with every type of condiment.
3-populating the files with all the current food crate related items in ASRS.
4-Removing the duplicate crate from misc.dm and operations.dm

## Why It's Good For The Game
With this new category people and cook will clearly see where to look if they want food items.

## Changelog

:cl:
add: Added a new category for ASRS : food.
add: new crates with (potato wheat tomato)*20 meat and fish  *5 and a crate with every type of condiment.
qol: regrouping all the food crate related item in the food category
/:cl:

### CODE REVIEW

**Comment by fira** [comment]
You need to:
 1. Include the new file in the DME
 2. Assign cost to a variable (cost = probably)
 3. Put the rest of the food in that category (eg ingredients crate)
But yeah it's a good idea

**Comment by Huffie56** [comment]
> You need to:
> 
>     1. Include the new file in the DME
> 
>     2. Assign cost to a variable (cost = probably)
> 
>     3. Put the rest of the food in that category (eg ingredients crate)
>        But yeah it's a good idea

thanks : 
1-i didn't saw any dme file linked to ASRS code. can you clarify what file i need to modify? i added the category in supplypack.dm
2-their is a cost assign via cost = RO_PRICE_WORTHLESS variable . not sure what you mean there...
3-I just did remove all the food from misc category  and one from operation.
please help for 1 and 2.

**Comment by fira** [comment]
well, most of these are missing the `cost =` part
also the new file mut be inside the main environment file `ColonialMarinesALPHA.dme`
i'd suggest running this through dreammaker and testing it to ensure it works

**Comment by Huffie56** [comment]
> well, most of these are missing the `cost =` part also the new file mut be inside the main environment file `ColonialMarinesALPHA.dme` i'd suggest running this through dreammaker and testing it to ensure it works

I am not sure when i manage to delete cost = . I corrected it.
I added the define in ColonialMarinesALPHA.dme fallowing the alphabetic order like it was.
Do i really need to run and test it via dreammaker? and if yes how?
thanks

File: code/datums/supply_packs/food.dm [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+	contains = list(
+					/obj/item/ammo_box/magazine/misc/mre,
+					/obj/item/ammo_box/magazine/misc/mre,
+					)
+	cost = RO_PRICE_WORTHLESS
+	containertype = /obj/structure/closet/crate/supply
+	containername = "\improper USCM MRE crate(x2)"
+	group = "Food"
+	
+	/datum/supply_packs/funfood
```
Comment (fira) [inline code/datums/supply_packs/food.dm:168]: indentation error here, there's one extra tab

**Review by fira — CHANGES_REQUESTED** [review]
The prefered way to test is to download VScode, get the game files, open the folder in VScode, then you can press F5 to build/launch the game and try things out. Don't hesitate to ask on discord if you need help with this

File: code/datums/supply_packs/food.dm (hunk 1 above)
Comment (Huffie56) [reply code/datums/supply_packs/food.dm:168]: i see thanks

**Comment by Huffie56** [comment]
> The prefered way to test is to download VScode, get the game files, open the folder in VScode, then you can press F5 to build/launch the game and try things out. Don't hesitate to ask on discord if you need help with this

I will try and come to discord if needed.
thanks

**Comment by Huffie56** [comment]
tested it's working
ASRS menu and ordering food

image

I changed the long ass description
Req result
image

**Review by fira — APPROVED** [review]

**Pull request merged**